modules/apps/frontend-js/frontend-js-react-web/src/main/resources/META-INF/resources/js/render.tsx

```diff
@@ -88,7 +122,12 @@ export default function render(
 					 * can be found.
 					 */
 					try {
-						ReactDOM.unmountComponentAtNode(container);
+						if (USE_REACT_16) {
+							ReactDOM.unmountComponentAtNode(container);
```

Comment:
Got warnings related to using `ReactDOM.unmountComponentAtNode` with version 18. Found some occurrences in the portal. Does this mean that the owner of the component should take care of it?

![Screenshot from 2024-09-30 13-14-07](https://github.com/user-attachments/assets/87aeb4cd-641c-4c93-a84a-151ee7891578)